Source/diablo.cpp

```diff
@@ -126,10 +126,16 @@ std::vector<std::string> DebugCmdsFromCommandLine;
 #endif
 GameLogicStep gGameLogicStep = GameLogicStep::None;
 QuickMessage QuickMessages[QUICK_MESSAGE_OPTIONS] = {
-	{ "QuickMessage1", N_("I need help! Come Here!") },
+	{ "QuickMessage1", N_("I need help! Come here!") },
 	{ "QuickMessage2", N_("Follow me.") },
 	{ "QuickMessage3", N_("Here's something for you.") },
-	{ "QuickMessage4", N_("Now you DIE!") }
+	{ "QuickMessage4", N_("Now you DIE!") },
+	{ "QuickMessage5", N_("Heal yourself!") },
+	{ "QuickMessage6", N_("Watch out!") },
+	{ "QuickMessage7", N_("Thanks.") },
+	{ "QuickMessage8", N_("Retreat!") },
+	{ "QuickMessage9", N_("Sorry..") },

```

Comment:
Did you mean to put two periods after "Sorry" here?

Reply:
Yes, but I've changed it to single period since it may be more natural.